core/models/asset.py

```diff
@@ -19,6 +19,10 @@
 logger = logging.getLogger(__name__)
 
 
+def user_abstract(user, filename):
+    return f'user_{user.id}_{filename}'
```

Comment:
I'm not sure this function is returning the right thing when you call it while uploading the file in the incident_report model. 
The file I uploaded for example is labelled `user_None_WASH_OFFICER_ADVERT.pdf`. The original file name is `WASH OFFICER ADVERT.pdf`.
The object `user` argument is capturing the `AssetIncidentReport` object. I would recommend that you reimplement this to return a proper name.

Reply:
on it

Reply:
@RonKbS 
mind giving it a look

Reply:
It's now returning a `user_email_file_name` format 👍🏽